octopusdeploy/schema_package_reference.go

```diff
@@ -113,6 +113,10 @@ func expandPackageReference(tfPkg map[string]interface{}) *packages.PackageRefer
 		Properties:          map[string]string{},
 	}
 
+	if id, ok := tfPkg["id"]; ok {
+		pkg.ID = id.(string)
+	}
+
```

Comment:
Without that ☝️ , updating of built-in trigger resource would fail, new `id` will be generated every time when updating deployment process.